src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -517,7 +774,6 @@
 
 (re-frame/reg-event-fx
  :credentials/save-success
- global-interceptors
```

Comment:
sorry, why are we taking these all out again? Wouldn't it be better to have an interceptor chain we can later decide to add stuff to than not?

Reply:
So as you can see, I replaced the explicit calls to `global-interceptors` on each effect with two calls of `re-frame/reg-global-interceptor`. The reason was because some effects had `global-interceptors`, and some didn't, so I thought in the latter case we literally forgot to add them, and these are interceptors that are supposed to be applied to each and every effect, right?